aspnetcore/signalr/get-started-signalr-core.md

```diff
@@ -0,0 +1,106 @@
+
+  ![New Project dialog in Visual Studio](get-started-signalr-core/_static/signalr-new-project-choose-type.png)
+
+  The libraries that host SignalR server-side code are included in the project template. Install the client-side JavaScript separately with [npm](https://www.npmjs.com/).
+
+  ```console
+   npm install @aspnet/signalr
+  ```
+
+3. Copy the *signalr.js* from *node_modules\\@aspnet\signalr\dist\browser* to the *wwwroot\lib* folder in your project.
+
+## Create the SignalR Hub
+
+A hub is a class that serves as a high-level pipeline that allows the client and server to call methods on each other.
+
+1. Add a class to the project by choosing **File** > **New** > **File** and selecting **Visual C# Class**. 
+
+1. Inherit from `Microsoft.AspNetCore.SignalR.Hub`. The `Hub` class contains properties and events for managing connections and groups, as well as sending and receiving data.
+
+1. Create the `Send` method that sends a message to all connected chat clients. Notice it returns a `Task`, because SignalR is asynchronous. Asynchronous code scales better.
+
+  [!code-csharp[Startup](get-started-signalr-core/sample/Hubs/ChatHub.cs?range=7-14)]
+
+## Configure the project to use SignalR
+
+The SignalR server must be configured so that it knows to pass requests to SignalR.
+
+1. To configure a SignalR project, modify the `ConfigureServices` method of the application's `Startup` class by inserting a call to `services.AddSignalR`.
+
+  `services.AddSignalR` adds SignalR as part of the [ASP.NET Core middleware](xref:fundamentals/middleware/index) pipeline.
+
+1. Configure routes to your hubs using `UseSignalR`.
+
+  [!code-csharp[Startup](get-started-signalr-core/sample/Startup.cs?highlight=22,40-43)]
+
+## Create the SignalR client code
+
+1. Replace the content in *Pages\Index.cshtml* with the following code:
+
+  [!code-cshtml[Index](get-started-signalr-core/sample/Pages/Index.cshtml)]
+
+  The preceding HTML displays name and message fields, and a submit button. Notice the script references at the bottom: a reference to SignalR and *chat.js*.
+
+1. Add a JavaScript file to the *wwwroot\js* folder named *chat.js* and add the following code to it:
+
+  [!code-javascript[Index](get-started-signalr-core/sample/wwwroot/js/chat.js)]
+
+## Run the app
+
+1. Select **Debug** > **Start without debugging** to launch a browser and load the website locally. Copy the URL from the address bar.
+
+1. Open another browser instance (any browser) and paste the URL in the address bar.
+
+1. Choose either browser, enter a name and message, and click the **Send** button. The name and message are displayed on both pages instantly.
+
+  ![Solution](get-started-signalr-core/_static/signalr-get-started-finished.png)
+
+## Next Steps
+
+
+[Publish to Azure](xref:tutorials/publish-to-azure-webapp-using-vs)
```

Comment:
There are a few extra steps needed when publishing a SignalR app to Azure App Service. You need to manually enable WebSockets and ARR Affinity. Should we remove this link until we can have a proper "Publish a SignalR App to Azure" article?

Reply:
@anurse  Good idea, I forgot about the Websockets thing. I'll remove until we have something better to direct the reader to.